server/src/get-options.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+source /usr/share/bash-completion/bash_completion
```

Comment:
This is not found on OS X. Wondering if the is a more generic way of getting the bash completions. We do have access to the man/help page already. 🤔